### ISSUE TITLE
PolyPen: Unhandled Exception Caught!

### ISSUE THREAD

**Issue by garthhendy** [CGCookie/retopoflow#637]
Please tell us what you were trying to do, what you expected RetopoFlow to do, and what actually happened.
Provide as much information as you can so that we can reproduce the problem and fix it.
Screenshots and .blend files are very helpful.
Also, change the title of this bug report to something descriptive and helpful.
Thank you!

-------------------------------------

```
Environment:

- RetopoFlow: 2.0.0
- Blender: 2.79.0 master 2017-09-11
- Platform: Windows, 10, 10.0.17134, AMD64, AMD64 Family 23 Model 1 Stepping 1, AuthenticAMD
- GPU: NVIDIA Corporation, GeForce GTX 1080 Ti/PCIe/SSE2, 4.6.0 NVIDIA 411.63, 4.60 NVIDIA
- Timestamp: 2018-10-03 14:06:30.287689


Error Hash: 45778fa507a7d9fccea1af465f5c6f06


Trace:

- EXCEPTION (<class 'AttributeError'>): 'NoneType' object has no attribute 'co'
- .../rfmode\rftool_polypen.py
- 000 0581:draw_postpixel() self.draw_lines([nearest_vert.co, hit_pos])
- .../common\profiler.py
- 001 0174:wrapper() return fn(*args, **kwargs)
- .../rfmode\rfcontext_drawing.py
- 002 0137:draw_postpixel() self.tool.draw_postpixel()
```

**Comment by vxlcoder** [comment]
Thanks for reporting, @garthhendy!

I believe this issue should be fixed in the latest version of `release/2.0.1` branch.

**Comment by vxlcoder** [comment]
Similar to issue #635, I have tested all I could and was not able to reproduce this issue. I'm closing this issue, but feel free to reopen if it still happens. Thanks!
